test/unit/utils/jwt.spec.ts

```diff
@@ -709,24 +709,24 @@ describe('JwksFetcher', () => {
 
       return keyFetcher.fetchPublicKeys().then(() => {
         expect(https.request).to.have.been.calledOnce;
-        clock!.tick((ONE_DAY_IN_SECONDS - 1) * 1000);
+        clock!.tick((SIX_HOURS_IN_SECONDS - 1) * 1000);
```

Comment:
Time math is a bit strange here. It seems we are running assertions at:

00:00:00 (start; calledOnce)
05:59:59 (calledOnce)
11:59:59 (calledTwice)
17:59:58 (calledTwice)
23:59:58 (calledThrice)

Not really a problem. But just wanted to point it out.

Reply:
We are currently checking the boundaries at 6 hour steps. Should this be cleaned up a bit?

Reply:
We could also check, 0, 5.59, 6, 11.59, 12, 17.59, 18 etc.

Reply:
That's kind of what I expected to see. But it's not a major issue. Feel free to disregard or address in a future PR.